Doc/library/ssl.rst

```diff
@@ -194,6 +194,11 @@ purposes.
       :data:`PROTOCOL_TLS_SERVER` protocol instead of generic
       :data:`PROTOCOL_TLS`.
 
+   .. versionchanged:: 3.13
+
+      The context now uses :data:`VERIFY_X509_PARTIAL_CHAIN` and
+      :data:`VERIFY_X509_STRICT` in its default flags.
+
```

Comment:
I think you also need to mention these new flags further in the paragraph explaining what the default settings are.

Reply:
15c0313f95fc7f5c73683f81f12669d134b4c5a0